packages/sw-runtime-caching/src/lib/request-wrapper.js

```diff
@@ -140,7 +145,23 @@ class RequestWrapper {
     assert.atLeastOne({request});
 
     const response = await this.fetch({request});
-    if (response.ok || response.type === 'opaque') {
+
+    // .ok is true if the response status is 2xx. That's the default condition.
+    let cacheable = response.ok;
+    // If we have any cacheWillUpdate callbacks defined...
+    if (this.callbacks.cacheWillUpdate) {
+      // ...run throuh the callbacks one at a time, using reduce() to keep track
+      // of the previous result. We explicitly use reduce() here rather than
+      // some() since we want to make sure to run all of the callbacks, even
+      // if we already have a true result from one.
+      // This allows cacheWillUpdate callbacks to have side effects other, than
+      // just returning true or false.
+      cacheable = this.callbacks.cacheWillUpdate.reduce((previous, cb) => {
```

Comment:
When we discussed this we were going to enforce a single will cache behavior.

Having multiple just feels error prone.